src/Components/Incidents/index.js

```diff
@@ -53,10 +53,23 @@ class Incidents extends PureComponent {
     });
   };
 
+  isFilteredOut = (incident) => {
+    const { filters } = this.context;
+    if (filters.exclude) {
+      const excluded = filters.exclude
+        .split(',')
+        .some((filter) => incident.service.summary.includes(filter));
+      if (excluded) return false;
+    }
+    if (
+      filters.showOnlyOwnIncidents
+      && incident.last_status_change_by.summary !== this.context.currentUser.name
+    ) return false;
+    return true;
```

Comment:
You could simplify this one:

```suggestion
    return !(filters.showOnlyOwnIncidents
      && incident.last_status_change_by.summary !== this.context.currentUser.name);
```